### PR TITLE
Feature/11 navbar

### PR DESCRIPTION
**Pull request by Laroche-Nicolas**

### CODE REVIEW

File: src/components/QMenu/Example.vue [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<template>
+
```
Comment (nicolasbarb) [inline src/components/QMenu/Example.vue:2]: this is empty

**Review by nicolasbarb — CHANGES_REQUESTED** [review]

**Review by zaktolba — APPROVED** [review]

File: src/components/QMenu/Example.vue [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+</template>
+
+<script setup>
+
+</script>
```
Comment (ChxGuillaume) [inline src/components/QMenu/Example.vue:7]: delete that

File: src/App.vue [hunk 3]
```diff
@@ -3,5 +3,5 @@
 </template>
 
 <script setup>
-import Example from './components/QHelloWorld/Example.vue';
+import Example from './components/QNavbar/Example.vue';
```
[the remark this answers is not in the document]
Comment (ChxGuillaume) [reply src/App.vue:6]: https://youtube.com/clip/Ugkxzt6Rjk75WxMN2zF8-5VLy2D1p0A0film

**Review by ChxGuillaume — CHANGES_REQUESTED** [review]
some changes to apply

(see: https://youtube.com/clip/Ugkxzt6Rjk75WxMN2zF8-5VLy2D1p0A0film)

other way good job

**Review by zaktolba — APPROVED** [review]

**Review by nicolasbarb — APPROVED** [review]

**Review by ChxGuillaume — APPROVED** [review]

**Pull request merged**